### PR TITLE
update description of KSI

### PR DESCRIPTION
**Pull request by cwhanse**
generalize definition to remove discretization (better follows the solarforecastarbiter implementation).  Add a reference.

### CODE REVIEW

**Comment by wholmgren** [comment]
@lboeman @cwhanse if it's ok with you two, I suggest that @lboeman can make a PR to Cliff's branch that adds a References section to the side bar and fixes up the linking. It's not hard, but getting the ruby environment set up to test it is a little more complicated (or it was for me).

**Comment by lboeman** [comment]
> @lboeman @cwhanse if it's ok with you two, I suggest that @lboeman can make a PR to Cliff's branch that adds a References section to the side bar and fixes up the linking. It's not hard, but getting the ruby environment set up to test it is a little more complicated (or it was for me).

Can do. Is this ready for that?

File: metrics.md [hunk 1]
```diff
@@ -99,17 +99,15 @@ By this definition, a perfect forecast has a $$ R^2 $$ value of 1.
...
 
-$$ D_n(p) = \text{max}( | \text{CDF}_O(p) - \text{CDF}_F(p) | ) \enspace \text{for} \enspace p \in [ p_k, p_{k+1} ] $$
+$$ D_n(p) = \text{max}( | \text{CDF}_O(p) - \text{CDF}_F(p) | )$$
 
-where $$ p_k $$ is defined as $$ p_k = p_{\text{min}} + kd $$ for $$k = 0, 1, \dots, K $$ and $$ d = (p_{\text{max}} - p_{\text{min}}) / K $$.
-
-In practice, $$ K = 100 $$ is typical. A KSI value of zero implies that the CDFs of the forecast and observed values are equal.
+A KSI value of zero implies that the CDFs of the forecast and observed values are equal.
 
 KSI can be normalized as:
```
Comment (wholmgren) [inline metrics.md]: typo in paragraph below too... "as a stastical that tests"

**Comment by wholmgren** [comment]
> Can do. Is this ready for that?

Yes. We'll want to add more references in the future, but that can be a separate issue/pr. Just getting the first example worked out will make that easier.

**Comment by lboeman** [comment]
Should the references sidebar menu have links to each reference, or should there only be links in the citations e.g. [Espinar]

**Comment by wholmgren** [comment]
Let's keep it simple and only link to the section header in the sidebar. But the `[Espinar]` link in the body text should be able to go to the Espinar reference (obviously doesn't do much while there is only this single reference).

**Comment by lboeman** [comment]
@cwhanse could you rebase your patch-1 branch on master? I can't make a clean PR at the moment I think the current PR I made to your branch will create a merge conflict.

**Comment by wholmgren** [comment]
would it be easier to merge cliff's pr to master and then leland's pr to master? I didn't mean to waste everyone's time :/

**Comment by lboeman** [comment]
That'd work to.

File: metrics.md [hunk 2]
```diff
@@ -99,23 +99,21 @@ By this definition, a perfect forecast has a $$ R^2 $$ value of 1.
...
 
 $$ KSI (\%) = \frac{100}{a_{\text{critical}}} KSI $$
 
-where $$ a_{\text{critical}} = V_c (p_{\text{max}} - p_{\text{min}}) $$ and $$ V_c = 1.63 / \sqrt{n} $$. When $$ n \geq 35 , $$ the normalized KSI can be interpreted as a stastical that tests the hypothesis that the two empirical CDFs represent samples drawn from the same population.
+where $$ a_{\text{critical}} = V_c (p_{\text{max}} - p_{\text{min}}) $$ and $$ V_c = 1.63 / \sqrt{n} $$. When $$ n \geq 35 , $$ the normalized KSI can be interpreted as a statistical that tests the hypothesis that the two empirical CDFs represent samples drawn from the same population.
```
Comment (wholmgren) [inline metrics.md]: There were actually two typos: first misspelling, then grammar. I think we want to say "as a statistic that tests..."

Comment (cwhanse) [reply metrics.md]: haha and I introduced (and corrected) a third variant

**Pull request merged**